tests/PostInstallHook/README.md

```diff
@@ -0,0 +1,30 @@
+# Post-Install Hook Tests
+
+These tests ensure proper functionality of the post-install script that is executed after installing ddtrace from a package.
+
+The post-install script is expected to be at the following location to run the tests:
+
+```
+/src/ddtrace-scripts/post-install.sh
+```
+
+To run the tests from this directory:
+
+```bash
+$ docker-compose run --rm php bash
+# In the container...
+$ cd ~/datadog \
```

Comment:
I was trying to run the command above with no success. It seems to be hanging in my laptop. Anything else I should do? If not then I can investigate more why here it is not working, or so it seems.

Reply:
Hum... I'm not sure why that is. I'm running the following:

```bash
$ cd tests/PostInstallHook/ && docker-compose run --rm php bash
```

From there I'm able to copy/paste the command to run the tests. So not sure what the issue is. Perhaps running `docker-compose pull php` to get the latest image?

Reply:
That was probably the cause. Anyway, this is just related to my local env and does not happen on CI, so not a blocker at all.